src/python/national_voter_file/us_states/mi/transformer.py

```diff
@@ -297,7 +299,7 @@ def extract_mailing_address(self, input_dict):
         ])
         mail_addr_dict = {}
         if address_str.strip():
-            usaddress_dict, usaddress_type = self.usaddress_tag(address_str)
+            usaddress_dict, usaddress_type = self.usaddress_tag(address_str)[0]
```

Comment:
Right, but this one is only using the second value `usaddress_type` and it's throwing an error in Travis that there are too many values to unpack because it's trying to unpack the first item in the tuple,`usaddress_dict`.